### PR TITLE
chore: Updating Antrea version to 0.6.0

### PR DESCRIPTION
**Pull request by reachjainrahul**

### CODE REVIEW

**Comment by mboersma** [comment]
Could you `git commit --amend` and force-push this PR so the commit has a [conventional commit message](https://www.conventionalcommits.org/en/v1.0.0/#summary)? Something like:

```
chore: update Antrea to version 6.0.0
```

Otherwise the commit will be missed by our tools and won't show up in the CHANGELOG for a release. Thank you!

**Comment by reachjainrahul** [comment]
> chore: update Antrea to version 6.0.0

Done

File: parts/k8s/addons/antrea.yaml [hunk 1]
```diff
@@ -378,23 +440,29 @@ spec:
...
             path: /healthz
-            port: 443
+            port: api
             scheme: HTTPS
           initialDelaySeconds: 5
           periodSeconds: 10
           timeoutSeconds: 5
+        resources:
+          requests:
+            cpu: 200m
```
Comment (mboersma) [inline parts/k8s/addons/antrea.yaml:458]: These resource requests would ideally be part of the addon's configuration, not hard-coded here. Maybe that's a separate PR.

Comment (reachjainrahul) [reply parts/k8s/addons/antrea.yaml]: Done

**Comment by reachjainrahul** [comment]
@jackfrancis @mboersma Could you guys please merge the PR, if everything looks Ok?

**Comment by jackfrancis** [comment]
Tested out O.K., thank you @reachjainrahul!

**Pull request merged**